spec/features/incomes/creates_new_income.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+feature "creates new income" do
+  scenario "successfully" do
+    user = create(:user)
+    create(:budget, user: user)
+    log_in_as user
+
+    visit new_income_path
+    fill_in "income_name", with: "Salary"
+    select "1", from: "income_due_date_3i"
+    select "April", from: "income_due_date_2i"
+    select "2017", from: "income_due_date_1i"
+    select "monthly", from: "income_frequency"
+    fill_in "income_amount", with: 1000.00
+    click_on I18n.t("helpers.submit.income.create")
+
+    expect(page).to have_text(I18n.t("incomes.show.title", name: "Salary"))
+    expect(page).
+      to have_text(I18n.t(
+        "flash.actions.create.notice",
+        resource_name: "Income",
+      ),)
```

Comment:
A suggestion to change format to:
```
    expect(page).to have_text(
      I18n.t("flash.actions.create.notice", resource_name: "Income"),
    )
```
Because it is more readable

Reply:
Done in sha: ea7bed2